### DISCUSSION TITLE
Moons crescent rotation

### DISCUSSION THREAD

**Discussion by bjarkeandersen** [cosinekitty/astronomy#253] [Q&A] [answered]
I'm looking for information about the rotation of the moons crescent - i.e. during winter it's laying down and during summer it's standing (I think), but what is the degree of rotation? Anyone who knows where I find infomation about that?

**Comment by cosinekitty** [comment, marked as answer]
I think the [Camera demo](https://github.com/cosinekitty/astronomy/blob/master/demo/python/camera.py) is what you are looking for. That is the Python version. Here is sample output:

```
$ ./camera.py
USAGE: ./camera.py latitude longitude [yyyy-mm-ddThh:mm:ssZ]

$ ./camera.py 30.5 -82.7
Moon horizontal position: azimuth = 259.695, altitude = 67.053
Moon check: (1.00000000, 0.00000000, 0.00000000, 2022-10-18T13:53:30.149Z)
Sun vector: (0.15665780, 0.58980540, 0.78755802, 2022-10-18T13:53:30.149Z)
Tilt angle of sunlit side of the Moon = 53.170 degrees counterclockwise from up.
Moon magnitude = -9.66, phase angle = 99.18 degrees.
Angle between Moon and Sun as seen from Earth = 80.66 degrees.
```

The "tilt angle" is what you are describing.

The same demo program is written in C, C#, and JavaScript also.

**Reply by bjarkeandersen** [reply]
This was exactly what I was looking for - thank you so much for pointing me in the right direction.
